### ISSUE TITLE
Stuttering while scrolling

### ISSUE THREAD

**Issue by Retr0view** [darkroomengineering/lenis#93]
Hi, 
I've encountered an issue with lenis + cmsload on https://www.coinlanding.page/. When the page first loads and you start scrolling, there is stuttering (most noticeable on screens with lower refresh rates). However, once you reach the bottom of the list and no new items are loaded, the stuttering stops. 

all implemented code should be visible via inspect tools, my suspicion is that lenis has a hard time catching up whenever there is more content being loaded in the dom. once everything has been loaded this issue disappears until the page gets reloaded

**Comment by Retr0view** [comment]
i've just tested setting the cms load mode to render-all which loads all elemts into the dom at pageload and the stuttering persists until the page has been scroled all the way to the bottom. this makes me think it doesnt have anything to do with loading elemnts into the dom but i might be wrong

**Comment by clementroche** [comment]
It seems like your FPS are dropping significatively in some ways. Keep in mind that Lenis will run at the maximum FPS possible, less FPS = Lenis running slower = visible scrolling issues. You should debug code that's blocking your main thread. In conclusion, something in your code is heavy to render for the browser so it impacts your framerate, i don't think Lenis is the cause of your issue.

**Comment by drokk3** [comment]
I also had an issue with stuttering in Chromium browsers (Chrome, Edge on Windows). When I was scrolling the page it was working fine no matter how long I was scrolling up and down, but as soon as I went to my application on the second monitor and then went back to the webpage then the scrolling started to stutter.
I tried to cancel the animationframe on window.onblur and start it again on window.onfocus and it actually seems to fix the issue.

`
let af;
const scrollFn = (time) => {
  lenis.raf(time);
  af = requestAnimationFrame(scrollFn);
};
    
af = requestAnimationFrame(scrollFn);

window.onfocus = function() {
  af = requestAnimationFrame( scrollFn );
};
window.onblur = function() {
  cancelAnimationFrame( af );
};
`
